### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by NataliyaHn**

### CODE REVIEW

**Review by katrinfaran — APPROVED** [review]
well done.

**Review by YaroslavDonchuk — CHANGES_REQUESTED** [review]
review, fixes are required

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,64 @@ describe('getHumanAge', () => {
...
+  test('should return zeros if age of cat and dog is zero', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return zeros if age of cat or dog < 15', () => {
+    expect(getHumanAge(5, 11))
+      .toEqual([0, 0]);
+  });
+
```
Comment (djtyrf312) [inline src/getHumanAge.test.js:30]: you didn't use here test design technique like boundary values

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -7,4 +7,64 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(18, 21))
+      .toEqual([1, 1]);
+  });
+
+  test('should return 1 if age of cat and dog is 23', () => {
+    expect(getHumanAge(23, 23))
+      .toEqual([1, 1]);
+  });
+
+  test('should return 2 if age of cat and dog is 24', () => {
```
[the remark this answers is not in the document]
Comment (djtyrf312) [reply src/getHumanAge.test.js:46]: > age should not be 2 if the age of cat and dog is 24
> 16-24(including) = 2

I think you are mistaken here, 24 == 2

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
add a little bit more tests with boundary values

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -7,4 +7,64 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(5, 11))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1 if age of cat and dog = 15', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should return 1 if age of cat or dog on range of 15 - 23', () => {
```
[the remark this answers is not in the document]
Comment (NataliyaHn) [reply src/getHumanAge.test.js]: The name is changed

File: src/getHumanAge.test.js (hunk 1 above)
Comment (NataliyaHn) [reply src/getHumanAge.test.js:30]: Yes, I realize this is not a boundary values technique. But I use the equivalence class technique and in my opinion, it is appropriate here

**Comment by NataliyaHn** [comment]
I added the tests and ask you to approve this work

**Review by djtyrf312 — APPROVED** [review]
